backend/src/main/kotlin/outcobra/server/web/controller/ExamController.kt

```diff
@@ -18,7 +18,8 @@ class ExamController @Inject constructor(val examService: ExamService) {
         return examService.save(examDto)
     }
 
-    @GetMapping("/active/exam")
+    // TODO not really accurate naming imho
+    @GetMapping("/exam/active")
```

Comment:
If I remember correctly we agreed on puttying the target entity to the end of the api mapping
Therefore I would suggest something like /activeSemester/exam. Like that, the naming would be applicable to other endpoints

Reply:
Should we mind now? This will change anyway when we implement the semester chooser or at least we will revisit it.

Reply:
It's just inconsistent... I don't see how the change does any good... why don't you just revert it?
(if you do not want to correct it, that would be the best IMO)

Reply:
It makes the frontend service easier but I can change it.